### PR TITLE
playsinline for Android wechat

### PR DESCRIPTION
**Pull request by bingods**

### CODE REVIEW

File: src/players/FilePlayer.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ export class FilePlayer extends Component {
     if (playsinline) {
       this.player.setAttribute('playsinline', '')
       this.player.setAttribute('webkit-playsinline', '')
+      this.player.setAttribute('x5-playsinline', 'true')
```
Comment (CookPete) [inline src/players/FilePlayer.js]: Does this need to be `true` or can it be `''` like attributes above?

Comment (bingods) [reply src/players/FilePlayer.js]: '' is ok

Comment (CookPete) [reply src/players/FilePlayer.js]: Ok, let's change it to be consistent and then I'll merge.

```suggestion
      this.player.setAttribute('x5-playsinline', '')
```

**Comment by CookPete** [comment]
Thanks! 👍

**Pull request merged**